client/js/libs/handlebars/firstLetterUppercase.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
+
+module.exports = function(string) {
+	return string.charAt(0).toUpperCase() + string.slice(1);
+};
```

Comment:
I know this is a stupid-ass function, but do you mind adding a couple test cases for this?
Our handlebars helpers are actually the most tested area of the client so far lol (especially once #972 lakes it into `master`).